app/gosqs/send_message_test.go

```diff
@@ -0,0 +1,111 @@
+package gosqs
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/Admiral-Piett/goaws/app"
+	"github.com/Admiral-Piett/goaws/app/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSendMessage_MaximumMessageSize_Success(t *testing.T) {
```

Comment:
Would it be clearer to just call it `TestSendMessage_Success`?  `MaximumMessageSize` doesn't seem to factor in much?

Reply:
This test clearly sets `MaximumMessageSize` with a large value to succeed the sending.
<img width="684" alt="image" src="https://github.com/Admiral-Piett/goaws/assets/156628139/f793d665-a6d2-4a07-8fc8-efe99a8bf3b1">
But I think there is no problem to rename 👍